components/combobox/src/auro-combobox.js

```diff
@@ -682,7 +762,7 @@ export class AuroCombobox extends LitElement {
           this.input.value = inputValue;
 
           // If the value got set programmatically make sure we hide the bib
-          if (!this.contains(document.activeElement)) {
+          if (!this.contains(document.activeElement) && !this.bibtemplate.contains(document.activeElement)) {
```

Comment:
Add comment to explain that we are checking the input in the trigger as well as the bib for focus.